host.go

```diff
@@ -217,9 +206,9 @@ func (h *Host) StartDocker() error {
 
 	switch h.Driver.GetProviderType() {
 	case provider.Local:
-		cmd, err = h.GetSSHCommand("sudo /etc/init.d/docker start")
+		_, _, err = h.GetSSHCommand("sudo /etc/init.d/docker start")
```

Comment:
Can you explain the Go error in that gist?  It looked like it was trying to Run a nil command I believe.